scripts/import/import_all_sv_data.pl

```diff
@@ -1530,8 +1532,19 @@ sub parse_9th_col {
     
     # Phenotype
     if ($key eq 'phenotype') {
+      # Some phenotypes contain a comma in the description
+      # Replace the comma before splitting by comma
+      # Example: '46,XX Testicular Disorders of Sex Development'
+      if($value =~ /46,/) {
```

Comment:
Is there a way to catch other cases, or is `46` the only expected? It might not be trivial, maybe a regex to check for a phenotype made out of numbers?

Reply:
Until now `46,` is the only present in the file. I tried to replace all commas in the description but some phenotypes are separated by `,` and others by `, ` making the splitting more difficult.